setup.py

```diff
@@ -46,7 +47,7 @@
     pass
 
 # set default flags
-without_openmp = False
+without_openmp = True
```

Comment:
This I think should be handled by command line switches since using multiple threads as default seems sensible to me

Reply:
Agree.